### PR TITLE
Mikey/code reviews

### PR DESCRIPTION
**Pull request by mikeygillen**
Updated Code review with comments taken onboard.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,3 +130,20 @@ Overly complex structures and unnecessary code can be avoided by effective refac
...
 * Long parameter lists
 
+
+## Code Reviews
+### Evaluation Feedback process
+#### Use checklists
+Checklists are the most effective way to eliminate frequently made errors. Code review checklists also provide team members with clear expectations for each type of review and can be helpful to track for reporting and process improvement purposes.
+
+#### Establish a process for fixing defects found
+The best way to ensure that defects are fixed is to use a collaborative code review tool that allows reviewers to log bugs, discuss them with the author, and approve changes in the code.
```
Comment (chriskavanagh97) [inline README.md]: Can you look for a collaborative tool reccomendation

https://www.softwaretestinghelp.com/code-review-tools/

check that link for som examples

**Review by chriskavanagh97 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -130,3 +130,20 @@ Overly complex structures and unnecessary code can be avoided by effective refac
...
+
+#### Foster a positive code review culture
+To make code reviews successful, it is important that managers are able create a positive culture of collaboration and learning in peer review.
+While it´s easy to see defects as purely negative, each bug can actually be an opportunity for the team to improve code quality. Peer review also allows junior team members to learn from senior leaders and for even the most experienced programmers to break bad habits.
+
+#### [Read more here...](https://smartbear.com/learn/code-review/guide-to-code-review-process/)
+
+### Code Reviews
```
Comment (harrythomas1998) [inline README.md:164]: Is this a heading for a new section or is this the caption for the image?

Comment (mikeygillen) [reply README.md:164]: Read more here is a hyperlink to where i gathered most of the information for the above text and the Code Reviews is caption for image would you like it removed??

Comment (alinakazzaa) [reply README.md:164]: I would say rename it to something else or remove it

**Pull request merged**